### PR TITLE
add weekly changelog 2.264~2.266 & lts changelog 2.249.3

### PR DESCRIPTION
**Pull request by zhaoying818**
### 检查项（Submitter checklist）

- [ ] 已添加 Issue
- [x] 已阅读[翻译规范](https://github.com/jenkinsci/localization-zh-cn-plugin/blob/master/specification.md)
- [x] [原文地址](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)

### Desired reviewers

@jenkins-infra/chinese-localization-sig

### CODE REVIEW

File: content/_data/changelogs/weekly.yml [hunk 1]
```diff
@@ -7267,6 +7267,250 @@
...
+    - Wadeck
+    message: |-
+      内部: 删除 LoadStatistics 的内联资源。
+  - type: bug
+    category: internal
+    pull: 5041
+    authors:
+    - Wadeck
+    message: |-
+      内部: 更正对 <code>TokenUuidAndPlainValue</code>.的限制范围。
```
Comment (yJunS) [inline content/_data/changelogs/weekly.yml:7393]: `.的`多了个.吧？

File: content/_data/changelogs/weekly.yml [hunk 2]
```diff
@@ -7267,6 +7267,250 @@
...
+    message: |-
+      使全局工具的 Maven 配置与配置代码化插件兼容。
+  - type: rfe
+    category: rfe
+    pull: 4294
+    issue: 41891
+    authors:
+    - jsoref
+    message: |-
+      对资源根 URL 设置进行少量英语清除。
```
Comment (yJunS) [inline content/_data/changelogs/weekly.yml:7443]: 感觉这句话不是很好理解的样子

**Review by yJunS — CHANGES_REQUESTED** [review]

File: content/_data/changelogs/weekly.yml (hunk 1 above)
Comment (zhaoying818) [reply content/_data/changelogs/weekly.yml:7393]:
```suggestion
      内部: 更正对 <code>TokenUuidAndPlainValue</code> 的限制范围。
```

File: content/_data/changelogs/weekly.yml (hunk 2 above)
Comment (zhaoying818) [reply content/_data/changelogs/weekly.yml:7443]:
```suggestion
      对英文资源根 URL 设置进行少量清理。
```

**Review by yJunS — APPROVED** [review]
LGTM

**Pull request merged**